app/controllers/application_controller.rb

```diff
@@ -628,7 +633,7 @@ def preload_current_user_data
           current_user,
           scope: guardian,
           root: false,
-          navigation_menu_param: params[:navigation_menu],
```

Comment:
Did you intentionally drop this? Is it no longer needed?

Reply:
Yes it was removed in an earlier commit (which I now lost :sweat: ), it's not used anywhere else in the code.